src/hex-code.ts

```
@@ -0,0 +1,48 @@
+  return hex.toLowerCase();
+}
+
+export function hexToBinary(hex: string): string {
+  // validation input
+  if (!/^[0-9A-Fa-f]+$/.test(hex)) {
+    throw new Error("Invalid hexadecimal string");
+  }
+
+  let binary = "";
+  for (let i = 0; i < hex.length; i++) {
+    const decimal = parseInt(hex[i], 16);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)